aptos-move/block-executor/src/executor.rs

```diff
@@ -813,6 +877,72 @@ where
         }
     }
 
+    fn apply_delayed_field_writes_sequential(
```

Comment:
maybe `apply_output_...`? Seems like we do it not just for delayed fields

Reply:
Done.